project-management/collateral.md

```diff
@@ -80,3 +81,26 @@ All client projects should have open issues organized in a project board, prefer
 - **Done/closed**: Issues that have been merged and fixed
 
 Sometimes, project owners will keep a `Done` column in addition to a `Closed` column. In these cases, issues addressed by merged PRs should be moved into `Done` and remain open until sprint planning meetings, when we reflect on them and close them as a group. We typically follow this practice with larger projects like Dedupe.io that involve a substantial budget and time for reflection.
+
+## FreshBooks projects
+
+Time is tracked at DataMade using [FreshBooks](https://my.freshbooks.com). For each item in the scope of work, create [a new FreshBooks project](https://my.freshbooks.com/#/projects) with the following configuration:
+
+- **Name**: The name of the project and scope item. Follow the naming convention `Project title – Scope item title`, e.g., `Payroll Phase 3 – Meetings and Project Planning`.
+- **Client**: Add the name, organization, and email of the client contact.
+    - Repeat clients should already exist in FreshBooks. Type their name, organization, or email address into the corresponding field, and FreshBooks will autosuggest matching clients in the system.
+    - If your project is for a new client, you can add them to FreshBooks from the project creation screen. Simply fill in their information in the Client form, and they will be added to the system. If you aren't sure who the client contact is, consult a partner or lead developer.
+- **Project members**: Select all staff assigned to the project. Note, if a staff member is not assigned to the project, it will not show up in their list of projects to track time against.
+- **Project type**: Always select `Hourly Project`. **Do not use `Flat Rate Project`, as it will set all tracked time to non-billable.**
+- **Set hourly rates**: Under "Select a billing method", select "Team Member Rates". Team member rates are set based on position. Default rates have been set for each team member in FreshBooks, but can be altered based on project in this screen.
+- **Project services**: Select relevant services from the existing list. `Research`, `Development`, `Project Management`, and `Meetings` is a good starting point for most projects.
+- **Hourly budget**: Divide the cost of the scope item by $175 to estimate the number of hours budgeted for this scope item.
```

Comment:
@derekeder When I write scopes, I always start with hours and multiply by $175 before sending. Is this a team-wide practice, i.e., does this point make sense? Otherwise, we'll need to revise.

Reply:
yes, this is how we price out scopes, so we should do the same in freshbooks